### ISSUE TITLE
Is log_softmax function missing in the Line 39 of generator.py?

### ISSUE THREAD

**Issue by albertwy** [yala/text_nn#11]
I think log_softmax function should be added in Line 39 if we want to use the Gumbel Softmax.

`logits = F.log_softrmax(self.hidden(activ))`

If I'm wrong, please let me know. 
Thanks.

![image](https://user-images.githubusercontent.com/17995697/54492275-0d17c500-4900-11e9-8bee-addd81d35934.png)
